### PR TITLE
[improvement](metrics) Metrics add all rowset nums and segment nums

### PR DESCRIPTION
**Pull request by xinyiZzz**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Metrics add all rowset nums and segment nums

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/olap/tablet.h [hunk 1]
```diff
@@ -550,6 +551,14 @@ inline Version Tablet::max_version() const {
     return _tablet_meta->max_version();
 }
 
+inline uint64_t Tablet::segment_count() const {
+    uint64_t segment_nums = 0;
```
Comment (yiguolei) [inline be/src/olap/tablet.h:555]: need read lock?

Comment (xinyiZzz) [reply be/src/olap/tablet.h:557]: fix~

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.09 seconds
 load time: 522 seconds
 storage size: 17122531884 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230129142921_clickbench_pr_86529.html

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**